### DISCUSSION TITLE
File Manager Empty

### DISCUSSION THREAD

**Discussion by SOLTEK-007** [GetPublii/Publii#1513] [Help] [answered]
I moved to a new PC. I installed Publii on my new computer. My website files are saved to my OneDrive so I was able to open Publii, after editing the path in the app-config file to include the new Windows username. I did a sync to my hosting and all is working as it was before. However, when I go into Tools & Plugins > File manager, I see a message that says, "There are no files in the root directory." In all honesty, I don't remember if I ever used this tool in the past, as it's been over a year since I last updated the site, so I don't know if it was ever working. How can I fix this? Thanks.

**Comment by dziudek** [comment, marked as answer]
Hi,

File Manager contains additional files, so if you did not put anything to this tool, then it can be empty

**Reply by SOLTEK-007** [reply]
@dziudek I checked my site via SFTP and I noticed there is no .htaccess file. Can I upload one via the Publii File manager?

**Reply by dziudek** [reply]
@SOLTEK-007 - yes, you can add this file to the File Manager and then it will be uploaded with your website files
